app/Repositories/Three/SignupRepository.php

```diff
@@ -17,19 +17,19 @@ public function create($data)
         // Create the signup
         $signup = new Signup;
 
-        $signup->northstar_id = $data['northstar_id'];
+        $signup->northstar_id = auth()->id();
```

Comment:
It might make sense to send these along as parameters from the controller (so that we could use the repository for things that aren't HTTP requests, like for creating signups from a CSV or something).